tests/__fixtures__/ethereumSignTransaction.js

```diff
@@ -358,5 +358,54 @@ export default {
                 v: '0x2a',
             },
         },
+
```

Comment:
lets keep it compatible with fixtures from  https://github.com/trezor/trezor-firmware/blob/master/tests/device_tests/test_msg_ethereum_signtx_eip1559.py

Reply:
These tests use another mnemonic, so I can't use the same ones. I am just generating new test cases with the new mnemonic. Is that fine?

Reply:
ah, i get it, the are using `mnemonic_all`. so either you can create another file next to `__fixtures__/ethereumSignTransaction.js` let say `ethereumSignTransactionEip1559.js` or you can set mnemonic for single case like:

```
{
            description: 'no data eip 1559',
            mnemonic: 'mnemonic_all`,
            params: ...
```

Reply:
Okay, I can port the tests over if you prefer that 😄

Reply:
Done!